sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/TestSuiteBase.java

```diff
@@ -1045,9 +1045,10 @@ public static Object[][] simpleClientBuildersWithDirectTcpWithContentResponseOnW
     }
 
     @DataProvider
-    public static Object[][] simpleClientBuildersForDirectTcpWithoutRetryOnThrottledRequests() {
+    public static Object[][] simpleClientBuildersWithDirectTcpWithoutRetryOnThrottledRequests() {
```

Comment:
Since this now includes gateway mode as well, should we call this `simpleClientBuildersWithoutRetryOnThrottledRequests` ?

Reply:
make sense, will update

Reply:
updated